share/spice/github_languages/github_languages.js

```diff
@@ -0,0 +1,69 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_github_languages = function(api_result){
+
+        if (!api_result || !api_result.meta.status === 200) {
+          return Spice.failed('github');
+        }
+
+        // There's a next linkpage link provided by the api, this converts it
+        // to a link to the next page on the actual github search, so by clicking
+        // More on Github you automatically get taken to the next page of results
+        var sourceUrl = api_result.meta.Link[0][0].
```

Comment:
Some API results. Example query [`game maker language`](https://api.github.com/search/repositories?q=%20language%3Agame%20maker%20language&sort=stars&callback=ddg_spice_github_languages) do not return any link results meaning this will fail. You'll need to ensure the `api_result.meta.Link` property exists before using it.